### PR TITLE
endpoint added for updating article votes

### PR DESCRIPTION
**Pull request by philjacks**
- endpoint added for updating article votes
- tested with both positive & negative numbers
- errors 404 & 400 tested

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -1,64 +1,126 @@
...
+          title: "Living in the shadow of a great man",
+          author: "butter_bridge",
+          created_at: "2020-07-09T20:11:00.000Z",
+          topic: "mitch",
+          votes: 50,
+          body: "I find this existence challenging",
+        });
+      });
+  });
+
```
Comment (dappernerddesigns) [inline __tests__/endpoints.test.js:108]: Missing a test for the inc vote to not be an integer, and a test for the inc votes key to be missing.

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Please add additional testing. While your front end should stop those behaviours, as you know we can interact with apis through tools like insomnia which would cause issues if not robustly tested.

File: __tests__/endpoints.test.js (hunk 1 above)
Comment (philjacks) [reply __tests__/endpoints.test.js:108]: Sorted :)

**Review by dappernerddesigns — APPROVED** [review]
Fab, merge when ready.

**Pull request merged**